test/common/modules/shared_helper.js

```diff
@@ -249,6 +249,10 @@ define([
     expect(json1 === json2, 'JSON data contents mismatch.').to.be.ok;
   }
 
+  function randomString() {
```

Comment:
Not sure how simple it would be to access from the modules test suite but `Utils.cheapRandStr` is essentially the exact same function

Reply:
This is a merge gone wrong, the helper function already exists — will remove this copy.

Reply:
Removed.